sdk/storage/Azure.Storage.Common/src/StorageRequestFailedException.cs

```diff
@@ -73,15 +68,8 @@ public StorageRequestFailedException(Response response, string message, Exceptio
             : base(
                   response?.Status ?? throw Errors.ArgumentNull(nameof(response)),
                   CreateMessage(response, message ?? response?.ReasonPhrase, errorCode, additionalInfo),
-                  innerException)
+                  GetErrorCode(response, errorCode), innerException)
```

Comment:
Nit - should we also pass this into `CreateMessage` two lines up?

Reply:
You didn't do it for that parameter so I also didn't :)